Kitodo/pom.xml

```diff
@@ -178,8 +178,8 @@
             <artifactId>avalon-framework-impl</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.elasticsearch.client</groupId>
-            <artifactId>transport</artifactId>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
         </dependency>
```

Comment:
🛈 The ElasticSearch client came with an HTTP client that was used by the Query URL Import Module. Due to the special architecture with the module loader, the JAR (of the HTTP client) must be deployed with the Kitodo Core, otherwise the application will encounter an NoClassDefFoundError at runtime when trying to access a catalog.